libs/block-editor/src/lib/plugins/dot-bubble-menu.plugin.ts

```diff
@@ -67,7 +76,18 @@ export class DotBubbleMenuPluginView extends BubbleMenuView {
 
         // New Properties
         this.component = props.component;
+        this.componentChange = props.component.instance.changeTo;
+        this.chamgeHTMLElement = this.componentChange.elementRef.nativeElement;
+
+        // Subscriptions
         this.component.instance.command.subscribe(this.exeCommand.bind(this));
+        this.component.instance.toggleChangeTo.subscribe(this.toggleChangeTo.bind(this));
+
+        // Load ChangeTo Options
+        this.setChangeToOptions();
+
+        // We do not need only listen editor scroll event but all scroll on the website.
+        document.body.addEventListener('scroll', this.hanlderScroll.bind(this), true);
```

Comment:
Why here as well?

Reply:
The reason I decided to listen to the `scroll` event on the 'document' instead of the editor is that, in the [issue](https://github.com/dotCMS/core/issues/21897), the behavior is due to a scroll in the dialog not in the editor